.changeset/tame-keys-provide.md

```diff
@@ -0,0 +1,18 @@
+---
+'@primer/react-brand': minor
+---
+
+Use VideoPlayer component to render hosted videos
+
+```jsx
+<VideoPlayer title="HLS On Demand Demo" crossOrigin="anonymous">
+  <VideoPlayer.Source src="https://cdn.api.video/vod/vi36GGvd6PoTqViQLxBWwHjJ/mp4/1080/source.mp4" />
+  <VideoPlayer.Track
+    src="https://raw.githubusercontent.com/JoshBowdenConcepts/Subtitles-Example/main/sample.vtt"
```

Comment:
This is linking to your personal account, could you replace it with a placeholder / example please? (doesn't need to be a real url, relative is fine).

Reply:
This has been updated in the following [SHA](https://github.com/primer/brand/pull/373/commits/a15991caf8ad89311382ef5e27a02f6acfcc13c7)

Reply:
Checking in on this ☝️, it seems to be the same as before FYI. Do you need to push up the changes?

Reply:
@JoshBowdenConcepts A reminder that this still needs to be updated

![Screenshot 2023-08-16 at 15 30 15](https://github.com/primer/brand/assets/13340707/00338a87-0041-49bd-aacc-77799c73ec1b)

Reply:
This has been updated in a more recent SHA

Reply:
🤔 Looks the same to me. If you go to the diff view of the PR, the latest commits show this 👇 

![Screenshot 2023-08-16 at 16 35 23](https://github.com/primer/brand/assets/13340707/be272551-e9f4-40e7-b795-60932866e238)

Here's the url where i'm seeing this:

https://github.com/primer/brand/pull/373/files

Reply:
@JoshBowdenConcepts Could you please check the PR to see if the change is visible and confirm the push rather than updating the SHA? As a reviewer, I'd like to see the pushed changes.